### ISSUE TITLE
Format for outputted data

### ISSUE THREAD

**Issue by maneesha** [carpentries/amy-feeds#43]
Does it matter if data is outputted as `csv` or `json` for two-dimensional data?

**Comment by fmichonneau** [comment]
I'd like to keep everything as JSON to avoid having to have 2 files formats as CSV won't support non-rectangular data. Do you have a good use case for CSV?

**Comment by fmichonneau** [comment]
Yes, at least for now, I'd like to keep all the data feeds in the JSON format (GeoJSON is a type of JSON).
